libticables/trunk/CMakeLists.txt

```diff
@@ -38,14 +38,14 @@ set(PUBLIC_HEADERS
         src/export1.h
         src/timeout.h)
 
+if(WIN32)
```

Comment:
On the one hand, this is not equivalent to the autotools build definitions and code. Chances are that part of the problem lies in libticables/trunk/src/detect.cc and libticables/trunk/src/win32/detect.cc , which check for `__WIN32__` and `__MINGW32__`, and conditionally include C source files within other C source files.

On the other hand, it's likely that neither of the currently implemented methods for I/O port access works on modern Windows anymore, and I still don't have a working setup (a computer with real DB9 & DB25 ports, yet sufficiently high-end to run modern 64-bit Windows) for testing a more portable piece of code contributed by Jonimus years ago.

Reply:
To be fixed later, as part of #75.

Reply:
Yep